src/findfile.c

```diff
@@ -1483,29 +1658,24 @@ ff_clear(ff_search_ctx_T *search_ctx)
     while ((sptr = ff_pop(search_ctx)) != NULL)
 	ff_free_stack_element(sptr);
 
-    vim_free(search_ctx->ffsc_file_to_search);
-    vim_free(search_ctx->ffsc_start_dir);
-    vim_free(search_ctx->ffsc_fix_path);
-    vim_free(search_ctx->ffsc_wc_path);
-
     if (search_ctx->ffsc_stopdirs_v != NULL)
     {
 	int  i = 0;
 
-	while (search_ctx->ffsc_stopdirs_v[i] != NULL)
+	while (search_ctx->ffsc_stopdirs_v[i].string != NULL)
 	{
-	    vim_free(search_ctx->ffsc_stopdirs_v[i]);
+	    vim_free(search_ctx->ffsc_stopdirs_v[i].string);
 	    i++;
 	}
 	vim_free(search_ctx->ffsc_stopdirs_v);
```

Comment:
```suggestion
	VIM_CLEAR(search_ctx->ffsc_stopdirs_v);
```
and then get rid of the next line `search_ctx->ffsc_stopdirs_v = NULL;` ?

Reply:
Yep